threads/serializers.py

```
@@ -40,17 +40,59 @@ def to_representation(self, instance):
         representation["cover_image"] = ImageSerializer(instance.images.first()).data
         return representation
 
+
+
+
+
+
+
+
 class CommentSerializer(serializers.ModelSerializer):
```

Flake8 (7.2.0)

[error] too many blank lines (8)

(E303)